doc/api/url.md

```diff
@@ -771,10 +771,22 @@ console.log(myURL.href);
 
 #### `new URLSearchParams()`
 
+<!-- YAML
+added:
+  - v7.0.0
```

Comment:
The class was added in v7.5.0 and the constructor in v7.0.0 🤔 ?

Reply:
oops! my mistake

Reply:
I don't think those additions are adding much, it's already implied that if there are no history data, it inherits from the class, having redundant info only increases the risk to let a mistake pass through

Reply:
OK, so the nuance is that `require('url').URLSearchParams` only existed after 7.5.0 (https://github.com/nodejs/node/commit/021338dc6d4bcfdd300c2ead71138101c74b9067), but the actual class was added along the rest of the `URL` API (https://github.com/nodejs/node/commit/a8ece149e210ed477e1ee308bd052ee95b41d286). I wonder if any of our user would be sensible to the nuance – hopefully there are no v7.x users left so it doesn't matter anyway.

Reply:
It's true that this history info may be useless for users (it's quite impossible to see v7.x users left)

But it is required in some cases, like [browser-compat-data](https://github.com/mdn/browser-compat-data). And it is normal way to add history info for every single feature (especially for new features, like [web crypto](https://nodejs.org/docs/latest/api/webcrypto.html), [web stream](https://nodejs.org/docs/latest/api/webstreams.html), [worker threads](https://nodejs.org/docs/latest/api/worker_threads.html)). Also, parent feature and children feature are not always have the same history info, like in this PR.